File: wlxx-services/wlxx-auth/wlxx-auth-provider/src/main/java/com/xzg/wlxx/auth/config/security/auth/AuthenticationVo.java
package com.xzg.wlxx.auth.config.security.auth;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Builder;
import lombok.Data;

/**
 * @author XiaoZG
 */
@Data
@Builder
public class AuthenticationVo {

    @JsonProperty("access_token")
    private String accessToken;
    @JsonProperty("refresh_token")
    private String refreshToken;
}

File: wlxx-services/wlxx-auth/wlxx-auth-provider/src/main/kotlin/com/xzg/wlxx/auth/config/security/auth/AuthenticationVo.kt
package com.xzg.wlxx.auth.config.security.auth

import com.fasterxml.jackson.annotation.JsonProperty
import lombok.Builder
import lombok.Data

/**
 * @author XiaoZG
 */
@Data
@Builder
data class AuthenticationVo(
    @JsonProperty("access_token")
    var accessToken: String? = null,

    @JsonProperty("refresh_token")
    var refreshToken: String? = null
)
